### DISCUSSION TITLE
Wanting to buy a CB antenna and had a question.

### DISCUSSION THREAD

**Discussion by CryptoClown11** [egzumer/uv-k5-firmware-custom#229] [Q&A] [answered]
Would this antenna work with the UV-K5?

https://www.amazon.com/gp/aw/d/B093C3FVZR/ref=ox_sc_act_image_3?smid=A2V1J05CGJHIXH&psc=1

If so, do I need an adapter to attach it?

I'm kinda new at this and I'm loving your custom firmware on my radio. Looking forward to future updates ☺️.

**Comment by egzumer** [comment]
I have no idea. There are plenty of Telegram and Facebook groups that will be much better place to ask such questions. This place is a firmware repository, it is all about one particular software, not a hardware.

**Comment by Hunterfnet** [comment, marked as answer]
You need an adaptor SMA-female to BNC-female like this https://aliexpress.com/item/32889630175.html
Same your antenna with correct adaptor https://aliexpress.com/item/1005005931727782.html
I personally prefers this one with adapter appart https://aliexpress.com/item/1005006186925926.html , with the other antenna you can make a dissaster on the radio and .... on some eyes , good testing and hobby
